apps/web/pages/api/countrycode.ts

```diff
@@ -0,0 +1,20 @@
+import type { NextRequest } from "next/server";
+
+export const config = {
+  runtime: "experimental-edge",
+};
+
+export default async function handler(req: NextRequest) {
+  const countryCode = req.headers.get("x-vercel-ip-country") ?? "";
```

Comment:
I would strongly recommend using `x-vercel-ip-country-region` here ([docs](https://vercel.com/docs/concepts/edge-network/headers#x-vercel-ip-country-region)), as it's a closer match to the [country code standards](https://en.wikipedia.org/wiki/ISO_3166-2) used throughout the rest of our globalized codebase.

Reply:
Out of curiosity, I looked into how `libphonenumber-js` is mapping a country's code to a given format.

[This line](libphonenumber-js) confirms they base their region codes off of CLDR data, which gives me more confidence that `x-vercel-ip-country-region` is the best way to go here.

Reply:
@AaronPresley BasePhoneInput from react-phone-number-input requires 2 character country code which is returned from x-vercel-ip-country